Game/templates/Game/play.html

```diff
@@ -91,6 +91,7 @@ <h3>Room : {{ room.room_id }}</h3>
                             <div class="col-md-12">
                                     <h6>Player: {{current_player.player_name.title }}</h6>
                                     <h6>Score: {{current_player.score }}</h6>
```

Comment:
I think "score" and "streak" are too vague. Maybe it would be better if you replace the word "score" with "current score " and "streak" with "high score".